tests/src/test/scala/com/nvidia/spark/rapids/ProjectExprSuite.scala

```diff
@@ -61,6 +61,7 @@ class ProjectExprSuite extends SparkQueryCompareTestSuite {
             Array(StructField("id", IntegerType), StructField("name", StringType)))))),
         new Column(Literal.create(List(BigDecimal(123L, 2), BigDecimal(-1444L, 2)),
           ArrayType(DecimalType(10, 2)))))
```

Comment:
Could you update to use the same code style here instead of `selectExpr` ? Like
```Scala
lit(Array(null)),
lit(Array(Array(null)))
```

Reply:
It didn't work, I tried this before, it will throw exception, 

```
Unsupported component type class java.lang.Object in arrays;
org.apache.spark.sql.AnalysisException: Unsupported component type class java.lang.Object in arrays;
```

Reply:
This is Scala vs Spark types.  An `Array(null)` in Scala is an `Array[Null]`. But Spark does not map `scala.runtime.Null` to the SQL null type. This might be a bug in Spark, but apparently unless you are doing crazy tests very few people do things like this.